src/content/docs/boilerplate/federated-search.mdx

```diff
@@ -0,0 +1,461 @@
+
+## Overview
+
+This federated search approach involves:
+
+1. Creating a content index for your site's pages and content
+2. Building a custom search block that fetches both content and commerce data
+3. Implementing a tabbed UI to display results separately
+4. Handling search queries in parallel for optimal performance
+
+This approach keeps your content and commerce searches separate but presents them as a unified experience to the user.
+
+<Aside type="tip">
+**Note:** This approach is ideal when you want quick implementation, have straightforward search requirements, or want to maintain separate control over content and commerce search logic.
+</Aside>
+
+## Tutorial Steps
+
+### Step 1: Create the Content Index Configuration
+
+First, you need to configure a content index that will crawl and index your site's content pages. This is done by creating a `query.yaml` configuration file.
+
+   Create the configuration by making a POST request to your site's admin config endpoint:
+
+   ```bash
+   POST https://admin.hlx.page/config/[owner]/[repo]/[branch]/query.yaml
+   ```
+
+   Replace `[owner]`, `[repo]`, and `[branch]` with your site's GitHub details.
+
+   Here's a sample `query.yaml` configuration that indexes blog content:
+
+   ```yaml title="query.yaml"
+   indices:
+     content:
+       target: /content-index.json
+       exclude:
+         - '/blog/index'
+       include:
+         - /blog/**
+         - /pages/**
+       properties:
+         title:
+           select: head > meta[property="og:title"]
+           value: |
+             attribute(el, 'content')
+         image:
+           select: head > meta[property="og:image"]
+           value: |
+             match(attribute(el, 'content'), 'https:\/\/[^/]+(\/.*)')
+         description:
+           select: head > meta[name="description"]
+           value: |
+             attribute(el, 'content')
+         lastModified:
+           select: none
+           value: |
+             parseTimestamp(headers['last-modified'], 'ddd, DD MMM YYYY hh:mm:ss GMT')
+         robots:
+           select: head > meta[name="robots"]
+           value: |
+             attribute(el, 'content')
+         content:
+           select: main
+           value: |
+             textContent(el)
+   ```
+
+   <Aside type="note">
+   **Configuration details:**
+   - `target`: The URL path where the index JSON file will be published
+   - `include`: Glob patterns for pages to include in the index
+   - `exclude`: Paths to exclude from indexing
+   - `properties`: Metadata fields to extract from each page
+   </Aside>
+
+   After creating this configuration, the system will automatically generate a content index at `https://[your-domain]/content-index.json` whenever you publish pages.
+
+**Expected result:** You should be able to access your content index at the configured target URL and see a JSON array of indexed pages (after you publish the pages).
```

Comment:
Not sure about the "after you publish pages"  -- if i have already published pages, will I see an index for thos pages?

```suggestion
**Expected result:** After you publish pages, you can access your content index at the configured target URL and see a JSON array of indexed pages.
```